zbus/src/abstractions/async_drop.rs

```diff
@@ -10,6 +10,7 @@
 /// Hopefully this will be unnecessary [in the future][itf] when Rust gain an `AsyncDrop` itself.
 ///
 /// [itf]: https://rust-lang.github.io/async-fundamentals-initiative/roadmap/async_drop.html
+#[doc(hidden)]
```

Comment:
This is not supposed to be private at all. Elsewhere in the docs, you'll even find recommendation for using it.